packages/react-native-web/src/exports/StatusBar/index.js

```diff
@@ -7,16 +7,92 @@
  * @providesModule StatusBar
  * @flow
  */
-
 import { Component } from 'react';
 
+
+const {head} = document
```

Comment:
This needs to be able to run in a server-side context too. Use `canUseDOM` guard as done elsewhere

Reply:
Didn't know of that, fixed.